City-of-Helsinki/drupal-helfi-tyo-yrittaminen
# UHF-X: Update Drupal core to latest version and multiple other modules

How to test this PR:

1. Install the site with latest dev. Run all `make drush-cim` and `make drush-updb` commands.
2. Checkout this branch.
3. Run `make drush-updb && make drush-cr && make drush-cim && make drush-cr`. There should be few updates.
4. Test the site that it works correctly. There is multiple module updates as well.
